src/apps/common/storage.py

```diff
@@ -44,6 +45,17 @@ def _get_bool(app: int, key: int, public: bool = False) -> bool:
     return config.get(app, key, public) == _TRUE_BYTE
 
 
+def _set_uint8(app: int, key: int, val: int):
+    config.set(app, key, val.to_bytes(1, "big"))
```

Comment:
I used "big" because that's what I've found in other instances, but maybe we should put this in to a constant?